.github/workflows/continuous-deployment.yml

```diff
@@ -491,7 +491,7 @@ jobs:
             --analyticsWorkspaceId ${{ secrets.AZ_ANALYTICS_WORKSPACE_ID }} \
             --actionGroupResourceGroup ${{ secrets.AZ_ANALYTICS_RG }} \
             --actionGroupName ${{ secrets.AZ_ACTION_GROUP_NAME }} \
-            --branchHashId ${{ needs.build-info.outputs.environmentHash }}
+            --branchHashId ${{ needs.build-info.outputs.environmentHash || 'DOES_NOT_EXIST' }}
```

Comment:
Didn't this work with an empty branch has previously?

Reply:
This needs to be tested on main branch.

Reply:
The original error is because the script expects branchHashId to be passed in with an additional argument to the script. This just can't be an empty string. For the main deployment, 'DOES_NOT_EXIST' value will be ignored.